modules/trusted-firmware-m/Kconfig

```diff
@@ -265,6 +265,8 @@ config PM_PARTITION_SIZE_BL2
 config PM_PARTITION_SIZE_TFM
 	hex
 	prompt  "Memory reserved for TFM" if !TFM_PROFILE_TYPE_MINIMAL
+	default 0xFE00 if TFM_PROFILE_TYPE_MINIMAL && TFM_CMAKE_BUILD_TYPE_DEBUG && \
+		BOOTLOADER_MCUBOOT
```

Comment:
@gregersrygg This is a question on completeness.
We didn't add this configuration on purpose when we added the ASSERT, the thought was that PM_PARTITION_SIZE_TFM would could be set by users.

As we can see that this is annoying we should add the other remaining one.
`default 0x50000 if TFM_CMAKE_BUILD_TYPE_DEBUG` should have the same addition, adding a configuration for 
`default 0x4FE00 if TFM_CMAKE_BUILD_TYPE_DEBUG && BOOTLOADER_MCUBOOT`.

This makes this list grow a bit more.

The configuration for TFM_REGRESSION_S we can ignore this for.
Ideally we could have moved this to the prj.conf for the samples that need this, but since there is a sample in zephyr this is easiest to have here still.

Reply:
Oh, I missed that one. Added now.